src/kaleidoscope/device/dygma/raise/Hand.h

```diff
@@ -0,0 +1,138 @@
+/* -*- mode: c++ -*-
+ * kaleidocope::device::dygma::Raise -- Kaleidoscope device plugin for Dygma Raise
+ * Copyright (C) 2017-2019  Keyboard.io, Inc
+ * Copyright (C) 2017-2019  Dygma Lab S.L.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef ARDUINO_SAMD_RAISE
+
+#include <Arduino.h>
+#include "wire-protocol-constants.h"
+
+struct cRGB {
+  uint8_t r;
+  uint8_t g;
+  uint8_t b;
+};
+
+namespace kaleidoscope {
+namespace device {
+namespace dygma {
+namespace raise {
+
+#ifdef __SAMD21G18A__
+	void twi_init(uint16_t clock_khz); // allow to be called by the Raise hardware setup
+	void twi_disable(void);
+#endif
+
+const uint8_t PROGMEM gamma8[] = {
```

Comment:
How closely does this match keyboardio's gamma table? would we want to provide this somewhere else in the source?

Reply:
I _think_ it is exactly the same, but that's a hunch, will need to double-check.

Reply:
It is the same, but the Model01's table lives in `KeyboardioScanner`, a separate repo for now. I can merge that in to Kaleidoscope as `kaleidoscope::device::keyboardio::model01::Hand`, like I did with the Raise, and then I can lift out the gamma table to something like `kaleidoscope::driver::led::Gamma8`.